modules/ROOT/nav.adoc

```diff
@@ -1,41 +1,41 @@
 .{product}
```

Comment:
I think there needs to be a page between Data Migration and the first tool that compares the different tools.

Reply:
Please see the updated preview. I repurposed the components page and moved it to a new first section.